File: URI/1 - INICIANTE/C/1158 - SomaDeImparesConsecutivos3.c
#include <stdio.h>

int main(){
	int x, y, soma = 0, temp, zz, num;
	scanf(" %d", &num);
	while(num > 0){
		soma = 0;
		scanf(" %d %d", &x, &y);
		temp = y;
		zz = x;
		while(temp > 0){
			if(zz % 2 != 0){
				soma += zz;
				temp--;
			}
			zz++;
		}
		printf("%d\n", soma);
		num--;
	}
	
	return 0;
}

File: URI/1 - INICIANTE/Python/1158 - SomaDeImparesConsecutivos3.py
z = int(input())
while z > 0:
    soma = 0
    x, y = map(int, input().split())
    temp = y
    zz = x
    while temp > 0:
        if zz % 2 != 0:
            soma += zz
            temp -= 1
        zz += 1
    print(soma)
    z -= 1